x11-plugins/pidgin-telegram/metadata.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>ck+gentoo@bl4ckb0x.de</email>
+		<name>Conrad Kostecki</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<longdescription>
+		Telegram-purple is a libpurple protocol plugin that adds support for the Telegram messenger.
+	</longdescription>
+	<use>
+		<flag name="gcrypt">Use dev-libs/libgcrypt instead of dev-libs/openssl.</flag>
+		<flag name="icons">Install nice icons.</flag>
```

Comment:
Is there a use case for installing it without 'nice icons'? What happens then?

Reply:
Okay, after looking through it, I tend to remove the no_icons variant and always install icons.
If not disabled, it installs default icons into `/usr/share/pixmaps/`, which, according to the readme, pidgin will use.